### ISSUE TITLE
Bug Report: this is a question (not a bug report)

### ISSUE THREAD

**Issue by AlexTheGuy73** [FunkinCrew/Funkin#3939]
### Issue Checklist

- [ ] I have properly named my issue
- [ ] I have checked the Issues/Discussions pages to see if my issue has already been reported

### Platform

Newgrounds (Web/HTML5)

### Browser

None

### Version

0.5.3

### Description (include any images, videos, errors, or crash logs)

how do i export a fucking animation as texture atlas
i downloaded adobe flash cs4 and adobe animate but NONE OF THAT WORKED
i just need adobe cs4?
HOW DO I EXPORT IT AS TEXTURE ATLAS WITH ADOBE FLASH CS4
pleaseeeeeeeeeeeeeeeee
im making a really high effort mod and i will add playable characters, but the problem is that :/
please someone reply me, im panicking right now
![Captura de pantalla 2024-12-15 194619](https://github.com/user-attachments/assets/d946f892-6d1c-4d4f-889b-a8e63abd957a)
![Captura de pantalla 2024-12-16 172101](https://github.com/user-attachments/assets/3687d5bf-34e0-418a-bbaa-de309d535a50)

### Steps to Reproduce

1. wasting my time for 5 months trying to download adobe cs4
2. open the fla file of the pico/bf freeplay version
3. right click on the main symbol of the animation or idk
4. oh... there isn't a "export as texture atlas" option
5. START SHOCKING
6. download adobe animate for free (with piracy)
7. try to open the fla file... oh a error apeared
8. I CANT DEAL WITH THIS-
9. oh i can export the animation as swf and import it here-
10. nevermind, this shit is not working
11. HEARTH ATACK
12. SEARCHING IN GOOGLE "fnf github"
13. FNF GITHUB ISSUES..
14. and thats the reason of why im here.

**Comment by TheAwesomeZave** [comment]
Yeah, according to the Adobe wiki CS6 is the earliest version that could export to atlas
